tests/bootstrap.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+call_user_func_array(function ($rootDir) {
```

Comment:
Can we skip this and go directly using the autoloader as the bootstrap script? We can give PSR-4 and Composer for granted here.

Reply:
The idea behind the IIFE here is more about isolating scope and the variables in it, and this is just safer. With regard to taking Composer for granted, that's for sure, but checking first makes code even more portable, since the autoload script may reside in a root package of which this is a dependency. Because it's so easy to make this bit safer, I just do it everywhere, even if there's a very low risk factor.

So, if you confirm, of course I will remove this 😃